getting-started/master-your-ui/the-table-view.md

```diff
@@ -36,6 +36,59 @@ The **extended search** lets you search also within _reference fields_ of `belon
 _Extended search_ is **not the default** search because it is **slower**.
 {% endhint %}
 
+### Advanced syntax for search
+
+{% hint style="info" %}
+_Advanced syntax for search_ is only supported on nodejs agents `@forestadmin/agent` starting from version 1.37. 
+{% endhint %}
+
+Search supports advanced syntax to help you find exactly what you are looking for:
+
+* `-term` will search for records that do not contain `term` ;
+* `property:term` will search for records that contain `term` in the `property` field ;
+* `relation.childProperty:term` will search for records that contain `term` in the `childProperty` field of the `relation` relation ;
+* `term1 OR term2` will search for records that contain `term1` or `term2` ;
+* `term1 AND term2` will search for records that contain `term1` and `term2` (equivalent to `term1 term2`);
+* `term1 AND (term2 OR term3)` will search for records that contain `term1` and either `term2` or `term3`.
+* `property:NULL` will explicitly search for records that contain the technical value `NULL` in the `property` field ;
+* `"multiple quoted words"` will search for records that contain the phrase `multiple quoted words` without splitting it into multiple terms ;
+
+All elements of the advanced syntax can be combined, for instance `property:term OR -term2 OR relation.childProperty:term3`.
+
+#### Searching on numbers
+
+Number fields support the following additional syntax:
+
+* `>42` will search for records that contain a number greater than `42` ;
+* `>=42` will search for records that contain a number greater than or equal to `42` ;
+* `<42` will search for records that contain a number lower than `42` ;
+* `<=42` will search for records that contain a number lower than or equal to `42`.
```

Comment:
I'm not sure why the 2nd line was a match 
<img width="1346" alt="image" src="https://github.com/ForestAdmin/documentation-user-guide/assets/19222576/4a7e06e9-f0f9-47dd-8bb8-9967d607bca9">